### PR TITLE
update graphviz links

### PR DESCRIPTION
**Pull request by nannany**
Update graphviz links.

### CODE REVIEW

File: compiler/rustc_graphviz/src/lib.rs [hunk 1]
```diff
@@ -1,21 +1,19 @@
-//! Generate files suitable for use with [Graphviz](https://www.graphviz.org/)
+//! Generate files suitable for use with [Graphviz](https://graphviz.org/)
```
Comment (JohnTitor) [inline compiler/rustc_graphviz/src/lib.rs:1]: Why did you change? I don't think it's necessary to fix the links below.

Comment (nannany) [reply compiler/rustc_graphviz/src/lib.rs]: I am not particular about adding "www" or not, so I changed it according to your policy.

Comment (JohnTitor) [reply compiler/rustc_graphviz/src/lib.rs]: I mean, unrelated diffs make it unobvious what's changed and fixed. From that point of view, I didn't request to add subdomains but remove these diffs.

Comment (JohnTitor) [reply compiler/rustc_graphviz/src/lib.rs]: Also, it'd be great if you could squash commits once it's done.

Comment (nannany) [reply compiler/rustc_graphviz/src/lib.rs]: I appreciate your advice.
And I squashed commits.

**Comment by lcnr** [comment]
thanks 

@bors r+ rollup
